src/models/Tag.js

```diff
@@ -0,0 +1,35 @@
+/**
+ * A model for tag in Leto-modelizer.
```

Comment:
Not consistent with the other models -> start with `Represent ...`

Reply:
Done